### DISCUSSION TITLE
Diminution de la taille de la map.

### DISCUSSION THREAD

**Discussion by cyril-marpaud** [Crownicles/Crownicles#1353] [Nouvelles idées]
Lorsque je me trouve dans une zone où le réseau n'est pas optimal, la map peut prendre jusqu'à 5 minutes pour s'afficher.
De manière générale, il est peu probable qu'il soit utile d'avoir une map en 8192*5488 qui pèse 25Mio (et tout le monde n'a pas un plan de données mobiles illimité).

**Comment by Pietagorh** [comment]
Faut savoir que Discord affiche pas les images en full qualité, donc c'est clairement pas 25 Mo

**Reply by BastLast** [reply]
Yup c'est la raison pour laquelle on n'a pas compressé nous même,  discord le fait déjà et avoir l'image en full qualité peut etre pratique dans certains cas...

**Reply by cyril-marpaud** [reply]
C'est cool d'avoir l'image en giga qualité quelque part yep, m'alors je sais pas comment discord compresse mais l'image générée est quand même clairement lourde.
Ou alors c'est le temps de compression qui est long, je ne connais pas les détails. Je suis seul à avoir des latences sur l'affichage de la map ?

**Reply by BastLast** [reply]
Non je vois très bien de quoi tu parles, mais faut voir si on peut réellement faire qqch pr ça sans avoir une image très très floue. Faut étudier si c'est solvable ou si c'est inhérents au fait d'avoir une image tout court...
